modules/gapi/include/opencv2/gapi/gstreaming.hpp

```diff
@@ -16,6 +16,7 @@
 #include <opencv2/gapi/util/optional.hpp>
 #include <opencv2/gapi/garg.hpp>
 #include <opencv2/gapi/streaming/source.hpp>
+#include <opencv2/gapi/gcommon.hpp>
```

Comment:
Is this really required by this particular header?

Reply:
i'll try to check it